### ISSUE TITLE
Duda nombre de usuarios en ventana de espera

### ISSUE THREAD

**Issue by MaxAl100** [IIC2233/Syllabus#240]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [x] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/4)
* [x] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues)
* [x] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía (https://github.com/IIC2233/syllabus/issues/1)
* [x] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello.
* [x] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (https://iic2233.github.io/contacto/)
* [x] Utilizaré un título descriptivo y llenaré correctamente esta plantilla
* [x] De ser necesario, solo colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado copia

### Duda
En la ventana de espera de ejemplo los nombres de los jugadores están completamente en mayúsculas. Esto significa que debemos hacer esto o simplemente fueron escritos estos nombres en mayúscula al entrar en la sala de espera?

**Comment by JadrinaDA** [comment]
Hola! Los nombres solo necesitan ser alfanuméricos y cumplir cierto largo, no importa si están en mayusculas o no. Tú puedes elegir el formato que los imprimes según como creas que se vea mejor.

**Comment by MaxAl100** [comment]
Yap, muchas gracias! Esto (como muestro los nombres en la pantalla) no tendría que detallarlo en el README o sí?

**Comment by JadrinaDA** [comment]
No, no afecta la correción asi que no tiene que ir en el README.
